Core/PixelEvent.swift

```diff
@@ -643,7 +643,17 @@ extension Pixel {
         case toggleReportDismiss
 
         case userBehaviorReloadTwiceWithin12Seconds
+        case userBehaviorReloadTwiceWithin24Seconds
+        case userBehaviorReloadAndRestartWithin30Seconds
+        case userBehaviorReloadAndRestartWithin50Seconds
         case userBehaviorReloadThreeTimesWithin20Seconds
+        case userBehaviorReloadThreeTimesWithin40Seconds
+
+        case siteNotWorkingShown
+        case siteNotWorkingDismiss
+        case siteNotWorkingDismissByNavigation
+        case siteNotWorkingDismissByRefresh
+        case siteNotWorkingWebsiteIsBroken
```

Comment:
I would say these were part of the experiment as well and were temporary pixels so it is best to remove them too!